spyder/plugins/editor/plugin.py

```diff
@@ -757,30 +751,34 @@ def on_completions_available(self):
         widget = self.get_widget()
         completions = self.get_plugin(Plugins.Completions)
 
+        widget.sig_after_configuration_update_requested.connect(
+            self._after_configuration_update
```

Comment:
```suggestion
            completions.after_configuration_update
```

I think this change will make things clearer and will avoid the need of a method (`_after_configuration_update`) that does exactly the same.